src/utils/mail.ts

```diff
@@ -0,0 +1,28 @@
+import nodemailer from "nodemailer";
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: 587,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USERNAME,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+export async function sendMail(
+  to: string,
+  subject: string,
+  text: string,
+  html: string
+) {
+  const mailOptions = {
+    from: process.env.SMTP_USERNAME,
+    to,
+    subject,
+    text,
+    html,
+  };
```

Comment:
zmieniłbym to na 

```ts
import type { SendMailOptions } from "nodemailer";

const mailOptions = {
  from: env.SMTP_USERNAME,
  to,
  subject,
  text,
  html,
} satisfies SendMailOptions;
```

żeby od razu była sprawdzana poprawność obiektu i w bonusie dostajemy autocompletion